tests/Model/ModelManagerTest.php

```diff
@@ -143,8 +146,15 @@ public function testInstantiateWithDeprecatedRegistryInterface(): void
         $this->assertSame($em, $this->modelManager->getEntityManager('x'));
     }
 
+    /**
+     * NEXT_MAJOR: Remove this test.
+     *
+     * @group legacy
+     */
     public function testSortParameters(): void
     {
+        $this->expectDeprecation('Method Sonata\DoctrineORMAdminBundle\Model\ModelManager::getSortParameters() is deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.');
```

Comment:
Please, place this expectation immediately before where the deprecation is intended to be triggered.

Reply:
done